swaglyrics/cli.py

```diff
@@ -108,8 +108,7 @@ def lyrics(song: str, artist: str, make_issue: bool = True) -> str:
 		# log song and artist for which lyrics couldn't be obtained
 		with open(unsupported_txt, 'a', encoding='utf-8') as f:
 			f.write(f'{song} by {artist} \n')
-			f.close()
```

Comment:
@aadibajpai this the only thing that had changed, it's because of this: https://github.com/SwagLyrics/SwagLyrics-For-Spotify/commit/25e4cc67bb7d22b5ad0f6834f39498899a63558f

Reply:
hahaha didn't know that "reviewed" stuff just meant to add a comment

Reply:
Yeah that's alright, I did it yesterday iirc. We don't need to close it since we use the `with` context